starboard/BUILD.gn

```diff
@@ -29,8 +29,7 @@ group("gn_all") {
     "//starboard/examples/hello_world:starboard_hello_world_example",
     "//starboard/examples/window:starboard_window_example",
     "//starboard/extension:extension_test",
-
-    # "//starboard/loader_app:app_key_files_test",
+    "//starboard/loader_app:app_key_files_test",
```

Comment:
FYI you should just be able to get rid of this after https://github.com/youtube/cobalt/commit/ff106d7c9ec1f6aa371a943d8144689d6df9354a#diff-f1d62fc45f45189fb4467d311d8511dacb54044877306b72dade7f4d99c63587 (and that'll resolve the merge conflict)